components/tensorflow-notebook-image/releaser/components/workflows.libsonnet

```diff
@@ -336,20 +331,21 @@
                 ],
               },  //dag
             },
-            buildImageTemplate("1.4.1", "1-4-1", "cpu"),
-            buildImageTemplate("1.4.1", "1-4-1", "gpu"),
-            buildImageTemplate("1.5.1", "1-5-1", "cpu"),
-            buildImageTemplate("1.5.1", "1-5-1", "gpu"),
-            buildImageTemplate("1.6.0", "1-6-0", "cpu"),
-            buildImageTemplate("1.6.0", "1-6-0", "gpu"),
-            buildImageTemplate("1.7.0", "1-7-0", "cpu"),
-            buildImageTemplate("1.7.0", "1-7-0", "gpu"),
-            buildImageTemplate("1.8.0", "1-8-0", "cpu"),
-            buildImageTemplate("1.8.0", "1-8-0", "gpu"),
-            buildImageTemplate("1.9.0", "1-9-0", "cpu"),
-            buildImageTemplate("1.9.0", "1-9-0", "gpu"),
-            buildImageTemplate("1.10.1", "1-10-1", "cpu"),
-            buildImageTemplate("1.10.1", "1-10-1", "gpu"),
+            //TODO tfma_version should be read from config.json. And so as the tf_package ...
+            buildImageTemplate("1.4.1", "1-4-1", "cpu", "disable_install"),
```

Comment:
Why do we need to specify the TFMA version as arguments to buildImageTemplate? Why isn't it being ready from config.json as the TODO mentions?

Reply:
Can we just rewrite build_image.sh to take in the config file?
https://github.com/kubeflow/kubeflow/blob/master/components/tensorflow-notebook-image/build_image.sh

Reply:
+1

Reply:
@jlewi  I will try it. As a fact, i am not familiar with scripts language. But  it will not make too much difficulty for me i think.

Reply:
@jlewi le I have a problem, why don't we use the file build_image.py in the build_image.sh？ It will Eliminate duplication and make code clear. If there is no special thoughts, i will use the python install scripts in build_image.sh

Reply:
Lets leave it as a TODO.

Reply:
It would be good to fix build_image.sh and not duplicate it; but this is blocking 0.3.1 so lets just get a quick fix to allow us to build the images by adding the library versions for TFMA and TF.DV as extra arguments and then fix build_image.sh or build_image.py in a follow on PR.